src/components/Book.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = (props) => {
+  const { title, author } = props;
+  return (
+    <div className="book">
+      <h1 className="book_title">{title}</h1>
+      <h1 className="book_author">{author}</h1>
+    </div>
+  );
+};
```

Comment:
- [ ] This component should have a button for removing books(it is not necessary to work yet though)